### PR TITLE
OSDOCS#6558: cert-manager Operator 1.11.1 is GA in 4.14

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.14 only

Issue: https://issues.redhat.com/browse/OSDOCS-6590

Link to docs preview: https://61494--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-technology-preview

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by xenolinux** [comment]
@xingxingxia Can you PTAL this PR?

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -898,6 +898,11 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
+|cert-manager Operator
+|General Availability
+|General Availability
+|General Availability
```
Comment (xingxingxia) [inline release_notes/ocp-4-14-release-notes.adoc:904]: Title is "Technology Preview tracker", but for cert-manager, all columns in https://61494--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-technology-preview show GA. Is it good to put such all-column-GA entry under a table called "Technology Preview tracker"?

Comment (swghosh) [reply release_notes/ocp-4-14-release-notes.adoc:904]: Similar view, +1.

Comment (xenolinux) [reply release_notes/ocp-4-14-release-notes.adoc]: @xingxingxia @swghosh Updated the PR. Added the section instead of entry in a table

**Comment by swghosh** [comment]
@xenolinux please check if we need a similar one for 4.13 stating that `cert-manager-operator 1.10 and 1.11` is generally available with **4.13**.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -117,6 +117,12 @@ Ingress Node Firewall Operator was made a technology preview in {product-title}
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-14-cert-manager-operator-1-11-1"]
+=== cert-manager Operator general availability
+
+{cert-manager-operator} 1.11.1 is generally available in {product-title} 4.14.
```
Comment (xingxingxia) [inline release_notes/ocp-4-14-release-notes.adoc:124]: This is right, but this is confusing easily making readers think the meaning appears it is GA starting with 4.14.

Comment (xenolinux) [reply release_notes/ocp-4-14-release-notes.adoc]: @xingxingxia Updated the section. This [1] information would suffice, right? Previous release notes are not mentioning the cert-manager Operator version, so I did not mention that here. 
[1]
```{cert-manager-operator} was introduced as a Technology Preview in {product-title} 4.11 and is now generally available in {product-title} 4.14.```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -117,6 +117,12 @@ Ingress Node Firewall Operator was made a technology preview in {product-title}
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-14-cert-manager-operator-1-11-1"]
+=== cert-manager Operator general availability
+
+{cert-manager-operator} was introduced as a Technology Preview in {product-title} 4.11 and is now generally available in {product-title} 4.14.
```
Comment (xingxingxia) [inline release_notes/ocp-4-14-release-notes.adoc:124]: Hmm, if I'm a new reader who does not know more about, this wording still make me not clear: is it TP or GA in 4.12 ~ 4.13? And the preview hides 1.11 now, this wording makes others not clear: 1.10 also GA in 4.14?
How about below:
```suggestion
{cert-manager-operator} 1.11.1 is now generally available in {product-title} 4.14 as well as 4.12 ~ 4.13.
```

Comment (swghosh) [reply release_notes/ocp-4-14-release-notes.adoc:124]: +1

Comment (xenolinux) [reply release_notes/ocp-4-14-release-notes.adoc]: Addressed

**Comment by xenolinux** [comment]
@xingxingxia I updated the PR. Can you please check if it looks good?

**Comment by xingxingxia** [comment]
/lgtm
Thanks!

**Review by sheriff-rh — APPROVED** [review]
LGTM!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 4]
```diff
@@ -117,6 +117,12 @@ Ingress Node Firewall Operator was made a technology preview in {product-title}
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-14-cert-manager-operator-1-11-1"]
+=== cert-manager Operator general availability
+
+{cert-manager-operator} 1.11.1 is now generally available in {product-title} 4.14 as well as {product-title} 4.13 and {product-title} 4.12.
```
Comment (xingxingxia) [inline release_notes/ocp-4-14-release-notes.adoc:124]: @xenolinux we know 1.11.4 is released. Need we update the hard code 1.11.1?

**Pull request merged**